### DISCUSSION TITLE
UpdateConnectionReferences and UpdateEnvironmentVariables

### DISCUSSION THREAD

**Discussion by h-karthik12** [microsoft/powerplatform-build-tools#703] [Ideas]
Separate task for updating connection reference and environment variable in power apps. you can refer @WaelHamze Power DevOps Automate tool. This would be helpful for automating the pipeline

**Comment by parvezghumra** [comment]
@h-karthik12 I agree that a separate task for ADO/Github Action would be useful for this purpose. However, it's already possible to achieve this behaviour as part of a normal solution deployment, using a Deployment Settings file, which I think you might already be aware of. Find more details [here](https://learn.microsoft.com/en-us/power-platform/alm/conn-ref-env-variables-build-tools).

**Reply by h-karthik12** [reply]
@parvezghumra Yes, This would work if your deploying a single solution. I was deploying using package deployer, which doesn't have a feature like deployment setting file in it.
